packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -135,6 +135,8 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
 
   private _renderCompositeLink(link: INavLink, linkIndex: number, nestingLevel: number): React.ReactElement<{}> {
     const isLinkSelected: boolean = this._isLinkSelected(link);
+    const isRtl: boolean = getRTL();
+    const paddingBefore: string = `${14 * nestingLevel}px`;
```

Comment:
I used `_indentationSize` defined at the top. I don't know why I put 14 instead, but I replaced it with `_indentationSize`.